mantidproject/mantid
# Fix GUI test builds with Makefile generator

**Description of work.**

Creates the specified output directories for cxxtest at CMake generate time. Some generators will not create these directories themselves.

**To test:**

* Run `cmake -G"Unix Makefiles"`
* Build `GUITests` - they should complete 

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
